overflow/app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,27 @@
+class AnswersController < ApplicationController
+  def new
+    if logged_in?
+      @answer = Answer.new
+    else
+      @errors = ["Please log in to post an answer"]
+      render :'sessions/new'
+    end
+  end
+
+  def create
+    answer = Answer.new(answer_params)
```

Comment:
You're not checking for logged_in here. Think about using a before_action to protect all your actions, then allow exceptions.